owasp-top10-2017-apps/a6/misconfig-wordpress/Makefile

```diff
@@ -8,7 +8,7 @@ COLOR_GREEN = \033[32m
 COLOR_RED = \033[31m
 
 PROJECT := A6 - Vulnerable Wordpress Misconfig
-PORT := 8000
+PORT := 10006
```

Comment:
Changing the A6 - Misconfig-Wordpress port breaks the app because of the WordPress inside, as its URL is actually pointing to `http://localhost:8000`.

I suggest leaving this app out, for now.

Reply:
Shall I revert all changes made to A6 ports, then?